typescript/sdk/src/metadata/chainMetadataTypes.ts

```diff
@@ -241,3 +241,9 @@ export function getChainIdNumber(chainMetadata: ChainMetadata): number {
   if (typeof chainMetadata.chainId === 'number') return chainMetadata.chainId;
   else throw new Error('ChainId is not a number, chain may be of Cosmos type');
 }
+
+export function mustGetReorgPeriod(chainMetadata: ChainMetadata): number {
```

Comment:
A convention we use in a few places across the SDK (like the MultiProvider and the get methods above) is for `get*` to throw and `tryGet` to return nullish. Suggest `getReorgPeriod`